examples/etc/config.yaml

```
@@ -32,12 +34,25 @@ poller:
         - 192.168.1.2
         - 192.168.1.3
         - 192.168.1.4
+        - localhost
+
   snmp_groups:
 
+      - group_name: Localhost-Test
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)